inst/using-the-dccvalidator-1kD.Rmd

```diff
@@ -0,0 +1,125 @@
+
+
+## Study and assay documentation upload
+
+Each study should have accompanying documentation in the VEO-IBD Synapse project. Here is an example of study [documentation in the Accelerating Medicines Partnership in Alzheimer's Disease portal](https://adknowledgeportal.synapse.org/#/Explore/Studies?Study=syn8391648){target="_blank"}, developed by Sage Bionetworks. 
+
+You can submit your documentation through the dccvalidator app on the Documentation page. There should be a study description for the whole study, and an assay description for each of the assays that was performed. These can be in a single file, or you can upload multiple files to the assay description
+section.
+
+#### How do I get access to a Staging folder to upload data? 
+
+With a new study, there may not yet be a **Staging folder** in the VEO-IBD Synapse project. Please contact us - [VEOIBD_SageAdmin@synaspse.org](mailto:VEOIBD_SageAdmin@synapse.org){target="_blank"}.
+
+
+## Data validation
+
+### Metadata requirements
+
+Each study should include metadata that would help a new researcher understand and reuse the data. In most cases, we will expect 4 files:
+
+1. **Individual metadata** describing each individual in the study
+   - Each row corresponds to a unique individual
+2. **Biospecimen metadata** describing the specimens that were collected
+3. **Assay metadata** describing the assay that was performed. If multiple assays were part of the study, there will be one assay file for each.
+4. A **manifest** listing each file that will be uploaded. Remember to include your metadata files in the manifest.
+
+Metadata file templates are [available in the VEO-IBD Synapse project](https://www.synapse.org/#!Synapse:syn20729790){target="_blank"}. 
+
+If you don't see a template for the assay(s) in your study, please send a request for a new schema to [VEOIBD_SageAdmin@synapse.org](mailto:VEOIBD_SageAdmin@synapse.org){target="_blank"}. We depend on your expertise to develop schemas that capture the most pertinent metadata!
+
+### Validating metadata
+
+The data validation portion of the app allows you to upload metadata files (as .csv) and the manifest (as .tsv or .txt) and view the results of a series of automated checks.
+
+Examples of the types of checks we perform are:
+- All required columns from the templates are present
+- Individuals and specimens have unique identifiers
+- Metadata terms conform to a controlled vocabulary, where applicable
+
+### Viewing data summary
+
+We also provide a summary of the files you have uploaded, showing the number of individuals, specimens, and files. We visualize the data in each column by its data type to help spot unexpected missing values.
+
+
+## Types of data to upload
+
+Currently, common data types are RNASeq, ATACSeq, ChipSeq and all single cell data. For common data types, **only fastq files are required**. For other data types, please provide **fastq** and **bam** files. 
```

Comment:
not sure what the common data types are for 1KD but this might need to be changed